<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CKTaskWorkModeTaskCodeSelectWidget</class>
 <widget class="QWidget" name="CKTaskWorkModeTaskCodeSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="ckWorkModeCombo"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>任务代号</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>点频</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="ckTaskCodeCombo1"/>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="ckPointFreqCombo1"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="ckTaskCodeCombo2"/>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="ckPointFreqCombo2"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="ckTaskCodeCombo3"/>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="ckPointFreqCombo3"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="ckTaskCodeCombo4"/>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="ckPointFreqCombo4"/>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="ckTargetRadioBtn1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="ckTargetRadioBtn2">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="ckTargetRadioBtn3">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="ckTargetRadioBtn4">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
